support-workers/src/main/scala/com/gu/acquisitions/AcquisitionDataRowBuilder.scala

```diff
@@ -54,14 +54,15 @@ import com.gu.support.zuora.api.ReaderType
 object AcquisitionDataRowBuilder {
   def buildFromState(state: SendAcquisitionEventState, requestInfo: RequestInfo): AcquisitionDataRow = {
     val commonState = state.sendThankYouEmailState
-    val (acquisitionProduct, amount) = productTypeAndAmount(commonState.product)
+    val productInformation = getProductInformation(commonState.product)
     val acquisitionTypeDetails = getAcquisitionTypeDetails(commonState)
     val printOptions = printOptionsFromProduct(commonState.product, commonState.user.deliveryAddress.map(_.country))
 
     AcquisitionDataRow(
       eventTimeStamp = DateTime.now(DateTimeZone.UTC),
-      product = acquisitionProduct,
-      amount = amount,
+      product = productInformation.product,
+      productName = productInformation.productName,
```

Comment:
it's a bit odd that we now have product and productName, where product is the UPPER_CASE ophan thrift enum that lumps all print/GW together, and productName is the more nuanced zuora catalog style one that separates all products out.

I think this is a side effect of the fact that this bus was originally just added as a layer before the old ophan call, so the format is pretty much identical to that?

I'd rather rename product to ophanProduct but that would be too much work.